integration-tests/package.json

```diff
@@ -31,6 +31,9 @@
     "@types/shelljs": "^0.8.8",
     "@typescript-eslint/eslint-plugin": "^4.26.0",
     "@typescript-eslint/parser": "^4.26.0",
+    "@uniswap/v3-core": "1.0.0",
+    "@uniswap/v3-periphery": "^1.0.1",
+    "@uniswap/v3-sdk": "^3.6.2-optimism-regenesis",
```

Comment:
Hmm is there a reason to use this specific version for the SDK?

Reply:
It's just the one that got automatically added. I'll remove the regenesis tag.